File: Neetcode/Golang/Arrays-Hashing/twoSum.go
package main

func twoSum(nums []int, target int) []int {
	seen := make(map[int]int)
	for i, val := range nums {
		lf := target - val
		if _, ok := seen[lf]; ok {
			return []int{seen[lf], i}
		} else {
			seen[val] = i
		}
	}
	return []int{}
}

File: Neetcode/Python/Arrays-Hashing/twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        seen = {}
        for i,val in enumerate(nums):
            lf = target - val
            if lf in seen:
                return [seen[lf],i]
            else:
                seen[val] = i
        return []